modules/aws_postgres/tf_module/rds.tf

```diff
@@ -21,18 +21,20 @@ resource "aws_rds_cluster" "db_cluster" {
   count                   = var.existing_global_database_id == null ? 1 : 0
   cluster_identifier      = "opta-${var.layer_name}-${var.module_name}-${random_string.db_name_hash.result}"
   db_subnet_group_name    = "opta-${var.env_name}"
-  database_name           = var.database_name
+  database_name           = var.restore_from_snapshot == null ? var.database_name : null
   engine                  = "aurora-postgresql"
   engine_version          = var.engine_version
-  master_username         = "postgres"
-  master_password         = random_password.pg_password.result
+  master_username         = var.restore_from_snapshot == null ? "postgres" : null
+  master_password         = var.restore_from_snapshot == null ? random_password.pg_password.result : null
```

Comment:
Setting this value during a snapshot restore will cause the password to be reset after the snapshot is restored.

Reply:
source?

Reply:
Same source as `master_username`